xml/bp_chap_choosing_dist_scheme.xml

```diff
@@ -18,63 +18,173 @@
+   the Internet, a disconnected setup in combination with RMT or SMT is
+   the recommended solution.  In this scenario, RMT or SMT stays in an
+   <quote>external</quote> network with a connection to &scc; and
+   synchronizes the software channels and repositories on a removable
+   storage medium. Then you separate the storage medium from RMT or SMT,
+   and mount it locally on your &susemgr; server to read the updated
+   data.
   </para>
 
-  <para>
-   The following procedure will guide you through using SMT.
-  </para>
+  <formalpara>
+   <title>RMT</title>
+   <para>
+    The successor of SMT and currently runs on the following systems:
+   </para>
+  </formalpara>
+  <itemizedlist>
+   <listitem>
+    <para>&sle; 15 (when available), 12 SP2, and 12 SP3</para>
+   </listitem>
+   <listitem>
+    <para>openSUSE Leap 42.2, Leap 42.3, and openSUSE Tumbleweed</para>
+   </listitem>
+  </itemizedlist>
+   <para>
+    RMT allows you to provision updates for all of your devices running
+    a product based on &sle; 12 SPx and later as well as openSUSE Leap.
+   </para>
+
+  <formalpara>
+   <title>SMT</title>
+   <para>
+    The predecessor of RMT and is no longer actively developed.  It runs on
+    &sls; 12 SPx and allows you to provision updates for products based
+    on &sle; 12 SPx and earlier.  You will still need it, if you want to
+    update &sle; 11 clients.
+   </para>
+  </formalpara>
+  
+  <sect2 xml:id="rmtool">
+   <title>&rmtool; (RMT) and Disconnected Setup (DMZ)</title>
+   <para>
+    The following procedure will guide you through using RMT.  It will
+    work best with a dedicated RMT instance per &susemgr;.
+   </para>
+   <para>
+    For more information, see <link
+    xlink:href="https://github.com/SUSE/rmt#offline-mode"/>.
+   </para>
+   <procedure>
+    <title>RMT: Fetching Repository Data from &scc;</title>
+    <step>
+     <para>
+      Configure RMT in the external network with SCC. For details about
+      configuring RMT with &sle; 12 SP3 or version 15 (when available),
+      see <link
+      xlink:href="https://github.com/SUSE/rmt#installation-and-configuration"
+      />.
+     </para>
+    </step>
+    <step>
+     <para>
+      Using RMT, mirror all required repositories.
+     </para>
+    </step>
+    <step>
+     <para>
+      Get the required JSON responses from SCC and save them as files at
+      the specified path (for example, <filename>/mnt/usb</filename>):
+     </para>
+<screen>&prompt.root;rmt-cli export data /mnt/usb</screen>
+    </step>
+    <step>
+     <para>
+      Export settings about repositories to mirror to the specified path
+      (in this case, <filename>/mnt/usb</filename>); this command will
+      create a <filename>repos.json</filename> file there:
+     </para>
+     <screen>&prompt.root;rmt-cli export settings /mnt/usb</screen>
+     <para>
+      Check <filename>repos.json</filename> and edit it to remove the
+      repositories you do not need.
```

Comment:
@keichwa While in theory it is possible to edit the JSON file, the documentation **absolutely must not** suggest this to the end-user. :scream: 

The user should choose which repositories they want to mirror using `rmt-cli repos enable` and `rmt-cli products enable` commands (i.e., `rmt-cli product enable sles/15/x86_64` to enable SLES15 repos). After enabling the repos, they will be written to `repos.json` file (which is used to transfer data from online RMT to offline RMT instance -- SUMA doesn't use this file) and the repos configured in this file will be exported into the export directory.

The user can also verify which repos are currently enabled with `rmt-cli repos list` command.